src/components/Contentful/StoryCard.vue

```diff
@@ -63,23 +83,39 @@ export default {
 			default: () => {},
 		},
 	},
+	data() {
```

Comment:
Minor, since they're referenced in `computed` below, these shouldn't be needed in `data` as well